web/src/components/molecules/Schema/types.ts

```diff
@@ -0,0 +1,35 @@
+export type Model = {
+  id: string;
+  name: string;
+  description: string;
+  key: string;
```

Comment:
Are these optional? In the modelcreationmodal PR they aren't required, but start with empty string values, so I am a little unsure if that is on purpose and/or desired.

Reply:
@KaWaite , I think key is required, and it is a backend edit that should be done as well